back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,354 @@
+- Threads pending 
+- Total number of customer messages 
+- Unread threads 
+- Closed threads 
+
+## CUSTOMER SERVICE
+
+### Settings wheel
+
+When clicking on it, two options are displayed:
+
+1) Show SQL query
+
+After clicking on it, the sql query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+2) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manage
+
+### Bulk actions drop-down
+
+This dropdown is enabled only if at least one message is selected, otherwise it's disabled.
+
+There is 5 options, see the improvement [#19197](https://github.com/PrestaShop/PrestaShop/issues/19197)
+
+1) Mark as "opened"
+
+After clicking on it, the status of the selected messages are changed for "opened" and a successfull message is displayed: "Update successful"
+
+2) Mark as "closed"
+
+After clicking on it, the status of the selected messages are changed for "closed" and a successfull message is displayed: "Update successful"
+
+3) Mark as "pending 1"
+
+After clicking on it, the status of the selected messages are changed for "pending 1" and a successfull message is displayed: "Update successful"
+
+4) Mark as "pending 2"
+
+After clicking on it, the status of the selected messages are changed for "pending 2" and a successfull message is displayed: "Update successful"
+
+5) Delete selection
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the selected messages are deleted, a successfull message is displayed "The selection has been successfully deleted."
+
+### Listing
+
+**The messages sent through the contact form in front-office are only listed here if the contact has the "Save messages?" option enabled.**
+
+This setting can be changed in Shop parameters > Contact > Contact, when editing the contacts.
+
+If the option is disabled for the contact that the customer chooses, the message is simply sent to the contact's email address, and is not stored in PrestaShop.
+
+#### Multistore
+
+Only the messages created in the selected store or store group (through the context) are displayed.
+
+Example, let's say I have the following confiuration:
+- Group 1
+  - Shop A
+  - Shop B
+- Group 2
+  - Shop C
+  
+If I select group 1 as context, then the messages created in Shop A and Shop B will be displayed.
+
+If I select Shop A as context, then the messages created in Shop A will be displayed.
+
+If I select all shops as context, then the messages created in Shop A, Shop B and Shop C will be displayed.
+
+#### Select all / unselect all checkbox
```

Comment:
Sorry, I can't find it on the mockup: is it two items of the bulk action dropdown, or do those buttons appear in some other place?

Reply:
I am talking of this checkbox allowing to select all / unselect all messages

![capture d'écran_2915](https://user-images.githubusercontent.com/13449658/102599329-9b485d00-411d-11eb-8e71-088adcd7a2e5.png)

Reply:
Oh, that's why it was not listed as part of the bulk actions drop-down. I was wondering why it was not listed above instead. :-)